lib/main.dart

```diff
@@ -68,13 +68,11 @@ class CircuitVerseMobile extends StatelessWidget {
             builder: (themeContext) => GetMaterialApp(
               title: 'CircuitVerse Mobile',
               localizationsDelegates: [
```

Comment:
If we are using generated `supportedLocales` then we should also be using generated delegates
```suggestion
              localizationsDelegates: AppLocalizations.localizationsDelegates,
```